### PR TITLE
Add moving-flow-DK story

### PR DESCRIPTION
**Pull request by sampettersson**

### CODE REVIEW

File: angel-data/moving-flow-DK.json [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+  "ifid": "2A84C71A-BD02-43AE-AA4A-DE446B4340C3",
+  "keywords": {
+    "backButton": "Back",
+    "externalInsuranceProviderConfirmTitle": "",
+    "selectActionSelectLabel": "Select",
+    "tooltipModalInformationLabel": "Information"
+  },
+  "lastUpdate": "2021-12-08T13:51:06.018Z",
+  "locales": [
+    "en_NO"
```
Comment (robinandeer) [inline angel-data/moving-flow-DK.json:12]: Why is this NO?

File: angel-data/moving-flow-DK.json [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+    },
+    {
+      "height": 100,
+      "id": "c91c87bc-0ec1-4f92-b70d-fe9cd623926c",
+      "left": 500,
+      "name": "address",
+      "selected": false,
+      "story": "1b738a20-0dfc-46b6-84a4-9a6613e4fa87",
+      "tags": [],
+      "text": "<Message>\n  {EMBARK_MOVING_DK_ADDRESS_MESSAGE_1}\n</Message>\n\n<Response>\n  {street}, {zipCode}\n</Response>\n\n<TextActionSet next=\"[[{EMBARK_MOVING_DK_CONTINUE_BUTTON}->sizeInput]]\">\n  <TextAction title=\"{EMBARK_MOVING_DK_ADDRESS_LABEL}\"\n              placeholder=\"{EMBARK_MOVING_DK_ADDRESS_PLACEHOLDER}\"\n              large=\"true\"\n              key=\"street\"\n  >\n  </TextAction>\n  <TextAction title=\"{EMBARK_MOVING_DK_POSTAL_CODE_LABEL}\"\n              mask=\"NorwegianPostalCode\"\n              placeholder=\"0123\"\n              key=\"zipCode\"\n  >\n  </TextAction>\n</TextActionSet>",
```
Comment (robinandeer) [inline angel-data/moving-flow-DK.json:148]: Should this be `mask="NorwegianPostalCode"`?

**Review by robinandeer — COMMENTED** [review]
Found some NO references

File: angel-data/moving-flow-DK.json (hunk 2 above)
Comment (sampettersson) [reply angel-data/moving-flow-DK.json]: I need to make use of the autocomplete later on anyways, but wanna figure out if stuff works at all, changed it to just PostalCode now 🤠

**Review by robinandeer — APPROVED** [review]

**Pull request merged**